l10n/en-US/viewer.properties

```diff
@@ -168,20 +168,28 @@ find_match_case_label=Match case
 find_entire_word_label=Whole words
 find_reached_top=Reached top of document, continued from bottom
 find_reached_bottom=Reached end of document, continued from top
-# LOCALIZATION NOTE (find_matches_count): The supported plural forms are
+# LOCALIZATION NOTE (find_match_count): The supported plural forms are
 # [zero|one|two|few|many|other], with [other] as the default value.
 # "{{current}}" and "{{total}}" will be replaced by a number representing the
 # index of the currently active find result, respectively a number representing
 # the total number of matches in the document.
-find_matches_count={[ plural(n) ]}
-find_matches_count[one]={{current}} of {{total}} match
-find_matches_count[other]={{current}} of {{total}} matches
-# LOCALIZATION NOTE (find_matches_count_limit): The supported plural forms are
+find_match_count={[ plural(n) ]}
```

Comment:
Having looked at the Firefox findbar implementation again, [see here](https://searchfox.org/mozilla-central/rev/7a91966c106a3dde924e040b4e5b72d33eb77bf7/toolkit/content/widgets/findbar.js#1133-1142), I'm no longer convinced that there's any attempting to format these numbers according to locale.
So it might be possible to simply this by remove the `toLocaleString()` calls from the JS code, and then change this to just `plural(total)` (and `plural(limit` for for the other string below).